source/help/messaging/sending-messages.rst

```diff
@@ -1,39 +1,48 @@
 .. _sending-messages:
 
-Sending Messages
+Messages
 ================
+ 
+Sending messages and replying to messages are important ways to keep conversation active with your team. You can edit and delete messages after you have sent them. You can also generate and share links to any message in Mattermost.
 
-Messages are composed in the center pane and reply messages are composed in the right-hand sidebar. Depending on your system configuration, you can edit and delete messages after they are sent. You can also generate and share links to any message in Mattermost.
-
-Sending a Message
+Sending Messages
 -----------------
 
-Write a message by typing into the text input box, then press **ENTER** to send it. Use **SHIFT+ENTER** to create a new line without sending a message. To send messages by pressing **CTRL+ENTER**, go to **Main Menu > Account Settings > Send messages on Ctrl + Enter**.
+Press **Enter** to send a message. To use paragraphs, press **Shift + Enter** to create a new line. If you want to, you can change the settings to send messages with **Ctrl + Enter** instead:
+
+-  **Main Menu > Account Settings > Send messages on Ctrl + Enter**
 
-Replying to a Message
+Replying to Messages
 ---------------------
 
-Reply to messages to keep conversations organized in threads. Click the reply icon next to any message text to open the right-hand sidebar where you can compose and send your reply. Replies are indented slightly in the center pane to indicate that they are child messages of a parent message.
+-  Click the **[...]** icon next to a message > click **Edit**. This keeps conversations organized in threads.
 
-When composing a reply in the right-hand sidebar, click the expand/collapse icon with two arrows at the top of the sidebar to make things easier to read.
+To adjust the width of the right-hand sidebar, click the expand/shrink icon with two arrows at the top of the sidebar.
+Replies are indented slightly in the center pane to indicate that they are child messages of a parent message.
 
-Editing a Message
+Editing Messages
 -----------------
 
-Edit a message by clicking the **[...]** icon next to any message that you’ve sent, then click **Edit**. Your System Admin may disable the **Edit** feature or set a time limit before messages become uneditable.
+-  Click the **[...]** icon next to a message that you have sent > click **Edit**.
+
+Your System Administrator may disable the **Edit** feature or set a time limit before messages become uneditable.
```

Comment:
See https://docs.mattermost.com/process/documentation-guidelines.html#word-usage-guidelines about the use of the word "may".

System Administrator > System Admin (I agree that System Administrator is better. However, PMs feel that System Admin is the preferred term)